agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -317,6 +317,9 @@ public static class PreviewConfiguration {
     public boolean disablePropagation;
     public boolean captureHttpServer4xxAsError = true;
 
+    public static EnabledByDefaultInstrumentation injectJavaScriptSnippet =
```

Comment:
you don't want to use `EnabledByDefaultInstrumentation`, which is for instrumentation.
instead you want to declare a new inner class called InjectJavaScriptSnippet, which has a public boolean enabled:

```Java
public static class InjectJavaScriptSnippet {`
  public boolean enabled;
}
```

and then inside PreviewConfiguration, have this 

```Java
public InjectJavaScriptSnippet  injectJavaScriptSnippet  = new InjectJavaScriptSnippet();
```